guides/customization/vnc.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Using VNC with Coder
+description: Learn how to use a VNC with Coder
+---
+
+This guide will show you how to set up a Virtual Network Computing (VNC) system
+in Coder. Coder does not have a specific set of VNC providers it supports. Coder
+will render the VNC, as long as it is installed on the image used to create the
+environment.
+
+## Step 1: Create a Dockerfile image with a VNC provider installed
+
+When creating the image, be sure to set the following variables:
+
+```console
+HOME=/home/coder
+USER coder
+PORT 1234
+```
+
+**Note:** Set `PORT` to the relevant port number for your VNC instance.
+
+Here is an
+[example image](https://github.com/cdr/enterprise-images/tree/ed66817dfa83e33132322c33bea653a8a5fdc057/images/vnc)
+that uses [noVNC](https://github.com/novnc/noVNC) and
+[TigerVNC](https://tigervnc.org).
```

Comment:
May want to note that noVNC is being used as the client and TigerVNC is the server

Reply:
Can I just say `noVNC` is the most confusing `VNC` client name lol